### ISSUE TITLE
[bug] `MOON_LOG=off moon bin pnpm` outputs tool instalation logs

### ISSUE THREAD

**Issue by iyefrat** [moonrepo/moon#1296]
**Describe the bug**

I want to use the output of `moon bin pnpm` in a github action to cache the pnpm store. This happens early on in the process and is the first time the tool is used, so I get installation logs. I tried to turn them off with `moon --log=off bin pnpm` and `MOON_LOG=off moon bin pnpm`, but they still show.

**Steps to reproduce**

**Expected behavior**

```
$ MOON_LOG=off moon bin pnpm
/Users/me/.proto/tools/pnpm/8.10.5/bin/pnpm.cjs
```

**Actual behavour**
```
$ MOON_LOG=off moon bin pnpm
▪▪▪▪ installing proto 0.29.1
▪▪▪▪ installing node 20.10.0
▪▪▪▪ installing pnpm 8.10.5
/Users/me/.proto/tools/pnpm/8.10.5/bin/pnpm.cjs
```

**Environment**

```
  System:
    OS: macOS 14.2.1
    CPU: (8) arm64 Apple M1 Pro
    Memory: 58.77 MB / 16.00 GB
    Shell: 5.9 - /bin/zsh
  Binaries:
    Node: 20.10.0 - ~/.proto/bin/node
    npm: 10.2.3 - /opt/homebrew/opt/node@18/bin/npm
    bun: 1.0.25 - /opt/homebrew/bin/bun
  Managers:
    Homebrew: 4.2.5 - /opt/homebrew/bin/brew
    pip3: 23.3.1 - /opt/homebrew/bin/pip3
    RubyGems: 3.0.3.1 - /usr/bin/gem
  Utilities:
    CMake: 3.28.1 - /opt/homebrew/bin/cmake
    Make: 3.81 - /usr/bin/make
    GCC: 15.0.0 - /usr/bin/gcc
    Git: 2.43.0 - /opt/homebrew/bin/git
    Clang: 15.0.0 - /usr/bin/clang
    Curl: 8.4.0 - /usr/bin/curl
  Servers:
    Apache: 2.4.56 - /usr/sbin/apachectl
  Virtualization:
    Docker: 24.0.7 - /usr/local/bin/docker
  IDEs:
    Emacs: 29.2 - /opt/homebrew/bin/emacs
    VSCode: 1.84.0 - /opt/homebrew/bin/code
    Vim: 9.0 - /usr/bin/vim
    WebStorm: 2022.2
    Xcode: /undefined - /usr/bin/xcodebuild
  Languages:
    Bash: 3.2.57 - /bin/bash
    Java: 18.0.2 - /usr/bin/javac
    Perl: 5.30.3 - /usr/bin/perl
    Python3: 3.11.7 - /opt/homebrew/bin/python3
    Ruby: 2.6.10 - /usr/bin/ruby
  Databases:
    SQLite: 3.43.2 - /usr/bin/sqlite3
  Browsers:
    Chrome: 120.0.6099.234
    Safari: 17.2.1
```
I'm on moon 1.20.1. The template should probably include a section for that.

**Comment by milesj** [comment]
Hrmm this is a tricky one. Those logs are just stdout, not actual `--log` messages, so they can't be turned off. They also happen outside of the `bin` command it self, so conditionally handling it is tough.

For now, you can use `proto bin pnpm`, since that's all moon does under the hood.

**Comment by milesj** [comment]
Fixed in v1.21
